docs/reference/commandline/create.md

```diff
@@ -57,6 +57,7 @@ Creates a new container.
       --privileged=false         Give extended privileges to this container
       --read-only=false          Mount the container's root filesystem as read only
       --restart="no"             Restart policy (no, on-failure[:max-retry], always)
+      --memory-swappiness=""     Tuning container's memory swappiness (accepted values 0-100)
```

Comment:
Tune a container's memory swappiness behavior. Accepts an integer between 0 to 100.